Latest Action Runs are Failing

Latest PR and build runs are failing with

```
=================================== FAILURES ===================================
____________________________ test_create_pyart_obj _____________________________

    def test_create_pyart_obj():
        try:
            obj = act.io.mpl.read_sigma_mplv5(act.tests.EXAMPLE_SIGMA_MPLV5)
        except Exception:
            return
    
>       radar = act.utils.create_pyart_obj(obj, range_var='range')

act/tests/test_utils.py:272: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
act/utils/data_utils.py:599: in create_pyart_obj
    import pyart
/usr/share/miniconda3/envs/act_env/lib/python3.8/site-packages/pyart/__init__.py:49: in <module>
    from . import io
/usr/share/miniconda3/envs/act_env/lib/python3.8/site-packages/pyart/io/__init__.py:13: in <module>
    from .sigmet import read_sigmet
/usr/share/miniconda3/envs/act_env/lib/python3.8/site-packages/pyart/io/sigmet.py:14: in <module>
    from ._sigmetfile import SigmetFile, bin4_to_angle, bin2_to_angle
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

>   ???
E   ValueError: numpy.ndarray size changed, may indicate binary incompatibility. Expected 96 from C header, got 88 from PyObject

pyart/io/_sigmetfile.pyx:1: ValueError
```

--- a/continuous_integration/environment_actions.yml
+++ b/continuous_integration/environment_actions.yml
@@ -6,7 +6,6 @@ channels:
 dependencies:
   - cartopy
   - pyproj
-  - numpy
   - scipy
   - matplotlib>=3.5.1
   - netcdf4
@@ -18,6 +17,7 @@ dependencies:
   - scikit-posthocs
   - pip
   - pip:
+    - oldest-supported-numpy
     - mpl2nc
     - metpy
     - arm-pyart

--- a/requirements.txt
+++ b/requirements.txt
@@ -1,6 +1,6 @@
 # List required packages in this file, one per line.
 pyproj
-numpy
+numpy>=1.22.3
 pandas
 matplotlib
 scipy

